primer/brand
# Image Component

## Summary

Creating an image component!

## List of notable changes:

- **Added** Image Component

## What should reviewers focus on?

- Making sure all test cases are thought through
- Making sure the rendered output is as expected

## Steps to test:

1. Open the Image component in CI-deployed preview environment
2. Go to Image story in Storybook
3. Verify that it behaves as described.

## Supporting resources (related issues, external links, etc):

- [API Exploration](https://www.figma.com/file/pYV9ln5Bkvh07lbzczaTkF/Primer-Brand---Image-Component-Exploration?node-id=0-1&t=CoiGn3tOAfI0QP9D-0)
- [Tracking Issue](https://github.com/github/primer/issues/2048)

## Contributor checklist:


## Reviewer checklist:
